src/main/scala/cromwell/engine/backend/jes/JesRuntimeInfo.scala

```diff
@@ -6,7 +6,7 @@ import cromwell.engine.backend.runtimeattributes.CromwellRuntimeAttributes
 import scala.collection.JavaConverters._
 
 object JesRuntimeInfo {
-  def apply(commandLine: String, runtimeAttributes: CromwellRuntimeAttributes): JesRuntimeInfo = {
+  def apply(commandLine: String, runtimeAttributes: CromwellRuntimeAttributes, preemptible: Boolean): JesRuntimeInfo = {
```

Comment:
While I understand why it is the way it is, it seems a little odd that preemptible is being passed in considering the other runtime attribute derived things aren't. Perhaps a comment that that particular value is being modified by other logic upstream would be in order.

Also while not at all strictly necessary this would be an opportunity to have a JesRuntimeInfo and PreemptibleJesRuntimeInfo ADT where the latter gloms on the .setPreemptible. While that'd make me smile don't feel pressured to do so here